packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.tsx

```diff
@@ -0,0 +1,163 @@
+/**
+ * @copyright Microsoft Corporation. All rights reserved.
+ *
+ * @file Screen Reader Alert component.
+ */
+
+import * as React from 'react';
+
+import { IScreenReaderAlertProps, defaultScreenReaderAlertProps, ReadingMode } from './ScreenReaderAlert.Props';
+import styles = require('./ScreenReaderAlert.scss');
+
+export interface IScreenReaderAlertState {
+  alertText: string;
+}
+
+/**
+ * The period of timeout used to clear the text after each rendering.
+ *
+ * More than 1000ms is critical for NVDA to read the text properly.
+ */
+const clearTextTime: number = 1000;
```

Comment:
const numbers like this should be CAPITALS_WITH_UNDERSCORES to differentiate from localVariables.